math/drgeo1/files/patch-geo-drgeo_dialog.cc.diff

```diff
@@ -0,0 +1,11 @@
+--- geo/drgeo_dialog.cc.orig	2003-10-27 11:17:25.000000000 +0100
++++ geo/drgeo_dialog.cc	2016-07-14 22:09:10.000000000 +0200
+@@ -38,7 +38,7 @@
+ // Used in the style dialod callback, I know it's ugly, but so easy
+ static drgeoFigure *selected_figure;
+ 
+-struct
++struct DrgeoDialogData
```

Comment:
Could you perhaps ask upstream to fix this?

Reply:
probably pointless; drgeo1 seems to be rather dead. I couldn't even find a SCM repository for it, just some old tarballs on sourceforge. Should the need for more changes arise it might be a good idea to put the source on github, though.

Reply:
OK, if the software is not really maintained, there's not much we can do.